### ISSUE TITLE
Just installed, but get error "AttributeError: 'WSGIRequest' object has no attribute 'ENABLED'":

### ISSUE THREAD

**Issue by rchrd2** [skoczen/django-seo-js#12]
Hello. This library looks promising and like a good time saver. I just installed it on my localhost, and I am getting this error.

"AttributeError: 'WSGIRequest' object has no attribute 'ENABLED'" referring to "File "/venv/lib/python2.7/site-packages/django_seo_js/middleware/useragent.py", line 18, in process_request"

Not sure what the context is, but maybe it needs `hasattr(request, 'ENABLED')` prepended to the if statement? ie:

```
        if not hasattr(request, 'ENABLED') or not request.ENABLED:
            return
```

**Comment by rchrd2** [comment]
I created a pull request for this. https://github.com/skoczen/django-seo-js/pull/13

**Comment by skoczen** [comment]
Thanks for the report - putting comments on the PR!

**Comment by skoczen** [comment]
This is now released in `0.3.1.`  Thanks for your contributions!
